README.md

```diff
@@ -1,10 +1,10 @@
-# pangraph
+# pangraph [![Build Status](https://travis-ci.org/tuura/pangraph.svg?branch=master)](https://travis-ci.org/tuura/pangraph)  
 
-[![Build Status](https://travis-ci.org/tuura/pangraph.svg?branch=master)](https://travis-ci.org/tuura/pangraph)  
-
-Pangraph is a Haskell library which offers for parsing and serializations for  
-network files. As well as conversions to other Haskell graph formats. An  
-example format is [GraphML](http://graphml.graphdrawing.org/).
+Pangraph is a Haskell library which offers for parsing and serializations for graph files. As well as conversions to other Haskell graph formats. An example format is [GraphML](http://graphml.graphdrawing.org/). A graphml file
+could for example represent the following graph, with vertices from `A` to `E` and connections, the edges, between them.
+![a-sample-graph](examples/graphs/graph-AE.png?raw=true)
```

Comment:
@thisiswhereitype Please use an vector (svg) image instead and scale it down -- it's too large at the moment!